vllm/inputs/preprocess.py

```diff
@@ -243,6 +295,14 @@ def _prompt_to_llm_inputs(
             multi_modal_data = tokens_content.get("multi_modal_data")
             mm_processor_kwargs = tokens_content.get("mm_processor_kwargs")
 
+            if multi_modal_data is not None and self._can_process_multimodal():
+                return self._process_multimodal(
+                    prompt_token_ids,
+                    multi_modal_data,
+                    mm_processor_kwargs,
+                    lora_request=lora_request,
+                )
```

Comment:
Should we add a fallback to remind user multimodal process is skipped if they inputs `multi_modal_data` but model doesn't haved a registered processor？

Reply:
Sure!

Reply:
Since none of the models use processor yet, I'll just log an info message.